flake.nix

```diff
@@ -71,6 +71,18 @@
             name = "hello_part_no_macros";
             partitions = [ "hello_part_no_macros" ];
           }
+          {
+            name = "redirect_stdio";
+            partitions = [ "redirect_stdio" ];
+            preRun = ''
+              tmpdir=$(mktemp -d -t redirect_stdio.XXXXXXXXXX)
```

Comment:
Should trap to remove the directory automatically on SIGTERM etc.

Reply:
Or even better just put the output somewhere in the current working directory, incase we want to inspect it. No reason to put it in tmpfs?